lib/build-asset-map-reader.js

```diff
@@ -24,3 +24,25 @@ function readJSON(assetMap) {
 		});
 	});
 }
+
+// Retries a function that returns a promise
+// The first argument is the function
+// The second argument is an array of ms to wait in between the tries
+function retry(fn, retries) {
+	return (...params) => {
+		return fn(...params).catch(err => {
+			return new Promise((resolve, reject) => {
+				if(retries.length > 0) {
+					let backoff = retries.shift();
+					setTimeout(_ => {
```

Comment:
This reads like you’re rewriting the custom `util.promisify` variant for `setTimeout`, and…

Reply:
`util.promisify` was introduced in Node 8. I therefore promisify by hand to keep backwards compatability to Node 6 😸 I agree though: I should promisify this separately 👍